src/pymor/core/gather_inputs.py

```diff
@@ -277,6 +277,8 @@ def load_mfdataset(data, rule_spec):
     for f in all_files:
         logger.info(f"  * {f}")
     mf_ds = xr.open_mfdataset(all_files, parallel=True, use_cftime=True, engine=engine)
+    if "time_counter" in mf_ds.dims and "time" not in mf_ds.dims:
+        mf_ds = mf_ds.rename({"time_counter": "time"})
```

Comment:
I think a more general solution would be nice here. For example, you could add something to the rule like `time_dimname`:


```suggestion
    if rule.get("time_dimename") in mf_ds.dims and "time" not in mf_ds.dims:
        mf_ds = mf_ds.rename({rule.get("time_dimename"): "time"})
```

Reply:
I have seen some variations in non-standard time label naming (`time1`, `time2`, `time-intv`, `time-fxc`, ...). One approach to fix this asking the user to provide the time dimension name via the setting on the rule. Another approach is to use the `preprocess` parameter of `xr.open_mfdataset` to automatically identify the time dimension irrespective of how to is label. There is already a function that can identify the time dimension correctly. see https://github.com/esm-tools/pycmor/blob/29376290973796c07d32426c708d3238367f8fb1/src/pycmor/core/time_utils.py#L34
the `preprocess` function can take advantage of `get_time_label` and then rename it or just set the attribute `{"axis": "T"}` to make xarray recognise as time dimension.

For now, I prefer rolling with the idea of user setting the time label in the rule.